src/Microsoft.OData.Core/UriParser/Resolver/UnqualifiedODataUriResolver.cs

```diff
@@ -47,27 +47,39 @@ public override IEnumerable<IEdmOperation> ResolveBoundOperations(IEdmModel mode
                 return base.ResolveBoundOperations(model, identifier, bindingType);
             }
 
-            return FindAcrossModels<IEdmOperation>(model, identifier, this.EnableCaseInsensitive)
-                .Where(operation =>
-                    operation.IsBound
-                    && operation.Parameters.Any()
-                    && operation.HasEquivalentBindingType(bindingType));
+            return FindOperations(model, identifier, this.EnableCaseInsensitive, true, bindingType);
         }
 
-        private static IEnumerable<T> FindAcrossModels<T>(IEdmModel model, String qualifiedName, bool caseInsensitive) where T : IEdmSchemaElement
+        private static IEnumerable<IEdmOperation> FindOperations(IEdmModel model, string qualifiedName, bool caseInsensitive, bool isBound = false, IEdmType bindingType = null)
         {
-            Func<IEdmModel, IEnumerable<T>> finder = (refModel) =>
-                refModel.SchemaElements.OfType<T>()
-                .Where(e => string.Equals(qualifiedName, e.Name, caseInsensitive ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal));
+            IList<IEdmOperation> results = new List<IEdmOperation>();
+
+            StringComparison strComparison = caseInsensitive ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
 
-            IEnumerable<T> results = finder(model);
+            GetOperationsForModel(model, qualifiedName, results, isBound, bindingType, strComparison);
 
             foreach (IEdmModel reference in model.ReferencedModels)
             {
-                results.Concat(finder(reference));
+                GetOperationsForModel(reference, qualifiedName, results, isBound, bindingType, strComparison);
             }
 
             return results;
         }
+
+        private static void GetOperationsForModel(IEdmModel model, string qualifiedName, IList<IEdmOperation> results, bool isBound, IEdmType bindingType, StringComparison strComparison)

```

Comment:
>qualifiedName [](http://example.com/codeflow?start=74&length=13)

this is not actually the qualified name -- it's the unqualified name (hence this logic, rather than invoking the logic in the base class in line 47)